src/styles/index.scss

```diff
@@ -1,3 +1,172 @@
-body {
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+
+* {
+  box-sizing: border-box;
+}
+
+.page {
+  font-family: "Roboto", sans-serif;
+}
+
+.page__body {
   margin: 0;
 }
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__image {
+    display: flex;
+
+    width: 160px;
+    height: 134px;
+
+    margin-bottom: 40px;
+  }
+
+  &__body {
+    display: flex;
+    flex-direction: column;
+    margin: 0;
+
+    &-title {
+      margin: 0;
+      margin-bottom: 4px;
+
+      font-size: 12px;
+      font-style: normal;
```

Comment:
`font-style: normal;` is a default browser rule. Remove from other selectors as well
```suggestion
```